dali/pipeline/operator/operator.cc

```diff
@@ -23,6 +24,10 @@ void OperatorBase::EnforceUniformInputBatchSize(const Workspace &ws) const {
   if (IsSplitOrMerge(spec_.GetSchema())) {
     return;
   }
+  // InputOperators have relaxed checks, since they actually create a batch.
+  if (IsInputOperator(spec_.GetSchema())) {
+    return;
+  }
```

Comment:
I don't think it's OK - ExternalSource worked before, so why wouldn't Input Operators? I don't think we can relax it for them.

Reply:
Good point, it shouldn't be relaxed for input ops, but for VideoInput. Changing.